packages/peregrine/lib/store/middleware/thunk.js

```diff
@@ -0,0 +1,4 @@
+import thunk from 'redux-thunk';
+
+export const extraArgument = {};
+export default thunk.withExtraArgument(extraArgument);
```

Comment:
Fortunately for us, `redux-thunk` has already considered our exact situation and [provides a method](https://github.com/reduxjs/redux-thunk#injecting-a-custom-argument) that makes it easy to provide our API client to every thunk.